### DISCUSSION TITLE
v2.8.11

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4740] [分享 / Share]
🐛修复
- 修复了搜索框中的 XSS 注入问题.
- 修复了搜索框在搜索页面不能同步关键词. (#4721)
- 修复 `评论区IP属地显示` 失效. (#4690)
- 在搜索框中删除搜索历史时将不再收起下拉菜单. (#4732)
- 修复功能反复开关时, 功能面板中的按钮重复出现. (#4731)
- 修复功能面板空间不足时出现折行. (#4730)
- 修复 `启用视频截图` 在番剧区失效. (#4719)

☕开发者相关
- Comment API 调整:
  - 增加对 `CommentArea` 销毁的监测.
  - `forEachCommentItem` / `forEechCommentArea` 参数统一支持 `added` 和 `removed` 回调.
  - `CommentReplyItem` / `CommentItem` 增加属性 `vueProps`.

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.11'>v2.8.11</a>.</em>

**Comment by Geeyun-JY3** [comment]
> - 修复功能面板空间不足时出现折行. (#4721)

这条对应的应该是 #4730 吧，#4721 的漏了没写。

**Comment by kethoven** [comment]
“自定义顶栏—稍后再看“视频数字不符。
例如 我实际有998个视频，当时顶栏上只显示958.
这个问题，出现有两三个版本。以前的版本没有问题
希望能够修复一下

**Reply by Geeyun-JY3** [reply]
我其实也有注意到相关情况，少了的应该是已失效的视频，而且没记错破站原生顶栏`收藏`里的`稍后再看`和[稍后再看页面](https://www.bilibili.com/watchlater/#/list)显示的数量好像还不一样，挺迷的，只是我还没认真验证过不能百分百肯定。

建议另开 issue。

**Reply by kethoven** [reply]
刚刚看了，确实有这个问题。这个数字个人感觉没有意义。
因为我更在意的是我还能够增加多少视频，而不是有多少有效的视频。如果你的数字显示不到1000，但是实际已经满了，加不了了，就很迷惑。

**Comment by kethoven** [comment]
> 我其实也有注意到相关情况，少了的应该是已失效的视频，而且没记错破站原生顶栏`收藏`里的`稍后再看`和[稍后再看页面](https://www.bilibili.com/watchlater/#/list)显示的数量好像还不一样，挺迷的，只是我还没认真验证过不能百分百肯定。
> 
> 建议另开 issue。

我就是因为issue必填内容太多，写到一半放弃了，所以才没有开issue。
描述一些吧，就是现在的顶栏稍后再看数字，跟稍后再看管理页面的一致，应该算是有效视频数量。但是这个数字没有播放页面的数字，有意义。我们一般不关注管理页面，反而更关注播放页面。因为可添加的数量，也更加的真实。符合习惯直觉。

**Comment by 921j** [comment]
能不能出一个禁止顶栏菜单鼠标悬浮自动弹出的功能？每次切换标签鼠标晃到那些“主站”、“直播”的字上面就会自动弹出一个长长的菜单，很烦人的说。。
![2024-05-22_231955](https://github.com/the1812/Bilibili-Evolved/assets/29940386/d5c85a5c-59dc-4e50-b0ba-24fb1412d23e)

**Reply by Geeyun-JY3** [reply]
嫌`自定义顶栏`这个`主站`的光标悬停出来的菜单烦人就直接把这个入口关掉吧，旁边的 logo 也能进主站首页。

另外功能请求建议另开讨论，发在这下面既不易检索分类，作者也有可能忽略或忘掉。

**Reply by 921j** [reply]
谢谢，我以为版本归纳都在一处呢。
